viite-UI/src/model/ProjectCollection.js

```diff
@@ -196,10 +197,11 @@
             startDate: result.project.startDate,
             publishable: false
           };
+          currentProject = result;
           projectErrors = result.projectErrors;
+          reservedParts = result.reservedInfo;
```

Comment:
we can start to use the existing methods setProjectErrors, setReservedParts and setFormedParts